frontstage/views/sign_in/sign_in.py

```diff
@@ -42,7 +42,7 @@ def login():  # noqa: C901
         bound_logger = logger.bind(email=obfuscate_email(username))
         bound_logger.info("Attempting to find user in auth service")
         try:
-            oauth2_token = oauth_controller.sign_in(username, password)
+            oauth_controller.sign_in(username, password)
```

Comment:
I think we should remove references to oauth, its confusing as the auth server doesn't use oauth at all